### DISCUSSION TITLE
Error occurred in 'EXTRACT METADATA'

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#12667] [Q&A]
### The bug

I have changed the time of some photos and videos using immich, but after making the changes, an xmp file has been added to the files. What is this xmp file for? I deleted it, and it always reports this error when updating metadata. What should I do if I can't find the file?

![image](https://github.com/user-attachments/assets/12758df0-8286-4130-b396-d538d58ac258)
![image](https://github.com/user-attachments/assets/c499df7a-6cc2-4007-a592-0644de7755c7)

### The OS that Immich Server is running on

Windows 11 Docker Desktop

### Version of Immich Server

V1.115.0

### Version of Immich Mobile App

-

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

1.
2.
3.
...

**Comment by bo0tzz** [comment]
The XMP file is what the changed metadata is written into. You shouldn't mess around in the Immich folders, because you will cause errors like this.

**Reply by pyccl** [reply]
I thought it was to change the time, but those were directly written into the file, but the result was not. At first, I wasn't sure what that file was for. I have already deleted those files now. How should we handle it?

**Comment by radh21301** [comment]
Try running the metadata extraction job on all assets. That could fix the issues caused due to the .xmp files deletion (I think) - but the changes you made a redefinitely lost with those xmp files.

The 'modify date is not a function' issues might be related to this:
https://github.com/immich-app/immich/issues/12634#issuecomment-2350951545

**Reply by bo0tzz** [reply]
You can also try running the 'sidecar metadata' job
